package-lock.json

```diff
@@ -5026,7 +5026,7 @@
       "dev": true,
       "requires": {
         "@sinonjs/formatio": "3.0.0",
-        "just-extend": "^4.0.0",
+        "just-extend": "^3.0.0",
```

Comment:
I don't think we need any `package-lock.json` updates for an additional `npm` script - @kochis unfortunately I think we're gonna keep running into this 😬